avogadro/qtgui/pythonscript.cpp

```diff
@@ -157,6 +157,7 @@ QByteArray PythonScript::execute(const QStringList& args,
     }
 
     realArgs.prepend("run");
+    realArgs.insert(1, "--as-is");
```

Comment:
I'm pretty sure the `--as-is` needs to be after the "run" and before the task part:
https://pixi.sh/dev/reference/cli/pixi/run/#examples